x-pack/test/functional/services/transform_ui/wizard.ts

```diff
@@ -75,6 +75,47 @@ export function TransformWizardProvider({ getService }: FtrProviderContext) {
       await testSubjects.existOrFail(selector);
     },
 
+    async parseTable(tableSubj: string) {
+      const table = await testSubjects.find(`~${tableSubj}`);
+      const $ = await table.parseDomContent();
+      const rows = [];
+
+      // For each row, get the content of each cell and
+      // add its values as an array to each row.
+      for (const tr of $.findTestSubjects(`~${tableSubj}Row`).toArray()) {
+        rows.push(
+          $(tr)
+            .find('.euiTableCellContent')
+            .toArray()
+            .map(cell =>
+              $(cell)
+                .text()
+                .trim()
+            )
+        );
+      }
+
+      return rows;
+    },
+
+    async assertTableResults(tableSubj: string, column: number, expectedColumnValues: string) {
+      await retry.tryForTime(2000, async () => {
+        // get a 2D array of rows and cell values
+        const rows = await this.parseTable(tableSubj);
+
+        // reduce the rows data to an array of unique values in the specified column
+        const uniqueColumnValues = rows
+          .map(row => row[column])
+          .flat()
+          .filter((v, i, a) => a.indexOf(v) === i);
+
+        uniqueColumnValues.sort();
+
+        // check if the returned unique value matches the supplied filter value
+        expect(uniqueColumnValues.join()).to.equal(expectedColumnValues);
```

Comment:
Currently, `expectedColumnValues` is a string with comma separated values and some work is done here to compare it that way. But I can see how things get tricky in case one entry is expected to contain a comma. I would suggest to make `expectedColumnValues` a `string[]` as `expect(arrayA).to.eql(arrayB)` works fine as well.

Reply:
Fixed in 8b93f03.